cypress/support/commands.js

```diff
@@ -38,6 +39,18 @@ Cypress.Commands.add('fmbGoToProfile', (profile = 1) => {
   });
 });
 
+Cypress.Commands.add('navigateTo', (route, state = null) => {
```

Comment:
nice abstraction!

Reply:
Yes! I think I will name it differently though to reflect that it accepts state.